cmd/rosa/main.go

```diff
@@ -105,3 +113,32 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func versionCheck() {
+	rprtr := reporter.CreateReporterOrExit()
+
+	currVersion, err := gversion.NewVersion(info.Version)
+	if err != nil {
+		_ = rprtr.Errorf("There was a problem verifying the current version of rosa.")
+		_ = rprtr.Errorf("Please be aware that you are possibly running an outdated version: %v", err)
```

Comment:
We avoid using "Please" because it can be mistranslated. I would change this to "You might be running on an outdated version. Make sure you are using the current version of ROSA."